lblod/app-digitaal-loket
# add lekp 2.1 migration

## ID
DGS-169

## Description

The subsidy LEKP 2.1 is updated with the following:
 - `indienen pact` has a new deadline of 05 december 2024 23:59 and so is available to create again
 - The period of the subsidy is changed to show `2023 - 2024` with `N.V.T` underneath
 - the `opvolgmoment` is closed with enddate 31/12/2023
 - The subsidy is now available for all 300 steden en gemeenten

## Type of change


## How to test

Visit subsidy module and verify the changes are made to `indienen pact`, `Periode` and `opvolgmoment`.
